packages/manager/modules/pci/src/projects/project/failover-ips/failover-ips.routing.js

```diff
@@ -102,6 +108,18 @@ export default /* @ngInject */ ($stateProvider) => {
             find(ipAddresses, { type: 'public' }),
           ),
         ),
+
+      availableRegionsForOrder: /* @ngInject */ (iceberg, projectId) =>
+        iceberg(`/cloud/project/${projectId}/region`)
+          .query()
+          .expand('CachedObjectList-Pages')
+          .execute()
+          .$promise.then(({ data: regions }) => {
+            return regions.map((region) => ({
+              ...region,
+              region: REGIONS[region.continentCode] || REGIONS.CA,
```

Comment:
Default is CA ?

Reply:
This is to handle cloud.RegionContinentEnum which has the following values: "ASIA"|"EU"|"NA"|"US".
So no default case :) (unless enum is changed)